File: ThirdLightBulb/ThirdLightBulb.java
package ThirdLightBulb;
public class ThirdLightBulb
{
    public static int Fibonacci(int val)
    {
        int start = 0;
        int next = 1;
        if(val == 1)
        {
            next = start;
        }
        else if (val == 2) {
            return next;
        }
        else{
        for(int i = 2; i < val; i++)
        {
            int temp = start+next;
            start = next; 
            next = temp;
        }
        }
        return next;

    }

    public static void main(String[] args)
    {
        int holder = Fibonacci(3);
        System.out.println(holder);

    }
}

File: ThirdLightBulb/ThirdLightBulb.py
class ThirdLightBulb:
    @staticmethod
    def fibonacci(val):
        start = 0
        next_val = 1

        if val == 1:
            next_val = start
        elif val == 2:
            return next_val
        else:
            for i in range(2, val):
                temp = start + next_val
                start = next_val
                next_val = temp

        return next_val

    @staticmethod
    def main():
        holder = ThirdLightBulb.fibonacci(3)
        print(holder)

ThirdLightBulb.main()
